puppeteer/spec/request-blocking.spec.ts

```diff
@@ -0,0 +1,51 @@
+import puppeteer, { HTTPResponse } from 'puppeteer';
+import { USER_LIST_SELECTOR, OUTPUT_RESPONSE } from '../data/selectors.json';
+import { BASE_URL, USER_LIST_URL } from '../data/constants.json';
+import { expect } from 'chai';
+
+describe('Request Blocking', () => {
+  describe('Blocking the request', () => {
+    let browser: puppeteer.Browser;
+    let page: puppeteer.Page;
+    let interceptedRequest: puppeteer.HTTPRequest;
+    let errorText: string;
+
+    before(async () => {
+      browser = await puppeteer.launch();
+      page = await browser.newPage();
+      await page.setCacheEnabled(false);
+      await page.goto(BASE_URL);
+      await page.setRequestInterception(true);
+
+      page.on('request', (request) => {
+        if (request.url() === USER_LIST_URL) {
+          interceptedRequest = request;
```

Comment:
You save the request into the variable `interceptedRequest`, but you do not use it anywhere.
This seems like an unnecessary step.

Reply:
![Alt Text](https://media.tenor.com/51L86w94i2UAAAAd/you-know.gif)